lighthouse-core/audits/seo/robots-txt.js

```diff
@@ -31,6 +31,28 @@ const DIRECTIVE_SAFELIST = new Set([
   'request-rate', 'visit-time', 'noindex', // not officially supported, but used in the wild
 ]);
 const SITEMAP_VALID_PROTOCOLS = new Set(['https:', 'http:', 'ftp:']);
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the robots.txt file on the page. This descriptive title is shown when the robots.txt file is present and configured correctly. */
+  title: 'robots.txt is valid',
+  /** Descriptive title of a Lighthouse audit that provides detail on the robots.txt file on the page. This imperative title is shown when the robots.txt file is misconfigured, which makes the page hard or impossible to scan via web crawler. */
+  failureTitle: 'robots.txt is not valid',
+  /** Description of a Lighthouse audit that tells the user *why* they need to have a valid robots.txt file. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'If your robots.txt file is malformed, crawlers may not be able to understand ' +
+  'how you want your website to be crawled or indexed.',
+  /** [ICU Syntax] Label for the audit identifying that the robots.txt request has returned a specific HTTP status code. */
+  displayValueHttpError: 'request for robots.txt returned HTTP status: {statusCode, number}',
```

Comment:
`displayValue...` for the display values var names makes more sense to me instead of prefixing with an error, all the displayValues prefix with `displayValue` and group them together as such.  IMO

Reply:
ok, maybe we could use a different word than "error" then?

Reply:
Changed this one, but I think we should keep `displayValueValidationError` because they literally are validation errors.